src/features/community/migrationSlice.ts

```diff
@@ -0,0 +1,70 @@
+import { PayloadAction, createSlice } from "@reduxjs/toolkit";
+import { AppDispatch, RootState } from "../../store";
+import { db } from "../../services/db";
+import { uniq } from "lodash";
+
+interface MigrationSlice {
+  links: Array<string>;
+}
+
+const initialState: MigrationSlice = {
+  links: [],
+};
+
+export const migrationSlice = createSlice({
+  name: "migration",
+  initialState,
+  reducers: {
+    setMigrationLinks: (state, action: PayloadAction<string[]>) => {
+      state.links = action.payload;
+    },
+  },
+});
+
+export default migrationSlice.reducer;
+
+export const addMigrationLink =
```

Comment:
add/reset can be reducers.

Reply:
I made the changes based on `src/features/community/communitySlice.ts` 

should I move `addMigrationLink`,`resetMigrationLinks`, including the db commands into the reducer section?

Reply:
I see. Yeah those favorites should be reducers too. Take a look at settingsSlice

Reply:
I'll work on these suggestions next week. Just keeping up with the conflicts for now

Reply:
@aeharding I've updated this, let me know if there's anything else in this context